frontend/app/protected/template.hbs

```diff
@@ -1 +1 @@
-{{liquid-outlet}}
+{{outlet}}
```

Comment:
liquid-outlet was included two times in the template tree and generated 4 div's as flexbox containers.